WireCompiler.podspec

```diff
@@ -1,11 +1,13 @@
 Pod::Spec.new do |s|
+  version = File.read('./README.md').match(/<version>([0-9.]+)<\/version>/).captures[0]
```

Comment:
Wanna do the same technique, but build.gradle instead? That'll always be accurate. Sometimes for the readme we skip alpha releases.